app/experimenter/kinto/README.md

```diff
@@ -0,0 +1,158 @@
+- To provide an additional security boundary from preventing experiments from being deployed without being approved by multiple parties
+
+For a collection that requires reviews where multiple records are modified, there is no way to promote a change to one record from `main-workspace` to `main` without also promoting all the others. For this reason, the workflows diagrammed below enforce that only a single change can ever be made to a collection within a single review cycle.
+
+### Experiment States
+
+An experiment has two distinct states:
+
+- Its lifecycle state
+- Its publish state
+
+![](diagrams/states.png)
+
+#### Lifecycle States
+
+- Draft: An experiment in draft has been created, and can be edited.
+- Preview: An experiment in preview can not be edited and is automatically published to the `nimbus-preview` collection
+- Live: An experiment in live can not be edited and is published to the collection corresponding to its target application after it has been reviewed in Experimenter and in Remote Settings
+- Complete: An experiment in complete can not be edited and is no longer published in Remote Settings
+
+#### Publish States
+
+- Idle: An experiment has no changes that require review or modification in Remote Settings
+- Review: An experiment has changes that require review in Experimenter before they can be published to Remote Settings
+- Approved: An experiment has changes that have been approved in Experimenter and must be published to Remote Settings
+- Waiting: An experiment has changes that have been published to Remote Settings and are awaiting further review in Remote Settings
+
+In theory an experiment can occupy any combination of these two states, but in practice it will only have a publish state other than Idle in Draft and in Live. Preview experiments can be modified in Remote Settings without any review, and Complete experiments will never be published to Remote Settings.
+
+## Workflows
+
+The following diagrams describe every interaction between Experimenter and Remote Settings. An experiment's state is denoted as `(lifecycle state, publish state)`. Any change which requires creating a changelog includes `+changelog`.
+
+The following actors are involved:
+
+- Experiment Owner: The user in Experimenter that creates and edits an experiment
+- Experiment Reviewer: The user in Experimenter that is not the owner that has permission to review the experiment in both Experimenter and Remote Settings
+- Experimenter UI: The frontend application surface that Experimenter users interact with
+- Experimenter Backend: The backend server that hosts the Experimenter API and database
+- Experimenter Worker: The celery worker that can interact with the Experimenter database and make API calls to Remote Settings
+- Remote Settings UI: The frontend application surface that Remote Settings users interact with and is only accessible via whitelisted VPN access
```

Comment:
re: "whitelisted",

I can't remember if we had a better word for this.
https://english.stackexchange.com/questions/51088/alternative-terms-to-blacklist-and-whitelist has a few suggestions, but "allow listed VPN access" sounds not grammatically correct; or at least sounds clumsy.